thriftbp/prometheus.go

```diff
@@ -255,3 +255,12 @@ func stringifyErrorType(err error) string {
 	}
 	return strings.TrimPrefix(fmt.Sprintf("%T", err), "*")
 }
+
+// We create this interface instead of using errors.go/Error because we want it
+//to always match the Baseplate.Error regardless of the version the user has put in their repository.
```

Comment:
nits:
1. add a space after `//`
2. don't capitalize `Baseplate` here because `baseplate.Error` means type `Error` from package `baseplate` (and we don't capitalize package names in go) :)

Reply:
updated